### DISCUSSION TITLE
Welcome to holidaily Discussions!

### DISCUSSION THREAD

**Discussion by iceu-bb** [TheWidlarzGroup/holidaily#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by danielmark0116** [comment]
Hi, this is awesome feature!

**Reply by iceu-bb** [reply]
Thank you 👍

**Comment by domiskoczylas** [comment]
Hope we'll do sth great :D
